editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2169,9 +2169,11 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 	if (pan_gesture.is_valid()) {
 		NavigationMode nav_mode = NAVIGATION_NONE;
 
-		NavigationMode change_nav_from_shortcut = _get_nav_mode_from_shortcut_check(NAVIGATION_LEFT_MOUSE, shortcut_check_sets, true);
-		if (change_nav_from_shortcut != NAVIGATION_NONE) {
-			nav_mode = change_nav_from_shortcut;
+		for (const ShortcutCheckSet &shortcut_check_set : shortcut_check_sets) {
```

Comment:
Pan gesture has no mouse button and shortcuts can be empty (e.g., orbit), so `_get_nav_mode_from_shortcut_check` can't be used here.